app/models/reference_history.rb

```diff
@@ -64,7 +64,7 @@ def reference_received
 
   def automated_reminder_sent
     chasers
-      .reference_request.or(chasers.follow_up_missing_references)
+      .reference_request.or(chasers.follow_up_missing_references).or(chasers.referee_reference_request).or(chasers.referee_follow_up_missing_references)
```

Comment:
Does `.or` result in a SQL `OR`? I suspect this isn't as performant as an `IN` list of chaser types if so, but perhaps AR does this under the covers.